sources/packages/api/src/database/entities/restriction.model.ts

```diff
@@ -46,12 +48,29 @@ export class Restriction extends RecordDataModel {
     nullable: false,
   })
   description: string;
+
   /**
-   * Maximum number of times the given restriction can be ignored
+   * Action type of the restriction
```

Comment:
- [ ]Please add the period.
- [ ] Can we have here the same comment added to the SQL column?